docs/cli/metrics.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: "Metrics"
+sidebar_order: 5
+description: "Learn how Sentry's command line interface can be used for sending metrics."
+---
+Metrics help you track and visualize the data points you care about, making it easier to monitor your application's health and identify issues.
+Learn more about [Metrics](/product/metrics).
+
+## Requirements
+- [Install](/product/cli/installation) Sentry CLI (min v2.32.0).
+
+## Configuration
+
+Sentry CLI uses your project's DSN to authorize sending metrics. To set it up, export the `SENTRY_DSN` environment variable:
+
+<SignInNote />
+```bash
+export SENTRY_DSN=___PUBLIC_DSN___
+```
+
+Alternatively, you can add it to your `~/.sentryclirc` config file:
+
+```ini {filename:~/.sentryclirc}
+[auth]
+dsn = ___PUBLIC_DSN___
+```
+
+## Sending metrics
+When emitting a metric that doesn't exist, the metric will be created for you. If it already exists, it will be updated.
```

Comment:
```suggestion
You can send metrics using the CLI's `send-metric` command. You need to provide at least the `--name` (or `-n`) parameter to identify the metric. When emitting a metric that doesn't exist, the metric will be created for you. If it already exists, it will be updated.
```